src/runtime/virtcontainers/utils/utils.go

```diff
@@ -306,15 +306,57 @@ func ConvertAddressFamily(family int32) pbTypes.IPFamily {
 	}
 }
 
+func reap(pid int) {
+	var _status syscall.WaitStatus
+	var _rusage syscall.Rusage
+	for {
+		_, err := syscall.Wait4(pid, &_status, unix.WNOHANG, &_rusage)
```

Comment:
You can pass `nil` instead of the unused `_status` and `_rusage` variables.

Reply:
If the process hasn't exited yet, wait4(WNOHANG) will just return 0 with no error and reap() will return without having reaped anything. You should just pass 0 to the flags.

Reply:
Fixed.